next/src/components/_global/cardsComponent/cardsComponent.tsx

```diff
@@ -0,0 +1,72 @@
+import styles from './cardsComponent.module.scss';
+import { type Cards } from '@/types/_global/Cards';
+import CentralizedHeading from '@/components/ui/centralizedHeading/centralizedHeading';
+import Img from '@/components/ui/Img';
+import Markdown from '@/components/ui/Markdown';
+
+export default function CardsComponent({ data: { centralizedHeading, cards } }: { data: Cards }) {
+  return (
+    <section className={styles.cardsComponent}>
+      <div className={` ${styles.componentWrapper} maxWidth`}>
+        <CentralizedHeading
+          data={centralizedHeading}
+          backgroundColor='var(--background)'
+        />
+        <div className={styles.cards}>
+          {cards.map(({ description, icon, title, pluses, plusesDescription }, i) => (
+            <div
+              className={styles.card}
+              key={i}
+            >
+              <header>
+                <Img
+                  className={styles.icon}
+                  data={icon}
+                />
+                <Markdown.h2 className={styles.title}>{title}</Markdown.h2>
+              </header>
+              <Markdown className={styles.description}>{description}</Markdown>
+              <Markdown className={styles.plusesDescription}>{plusesDescription}</Markdown>
+              {pluses.map((plus, i) => (
+                <div
+                  className={styles.plus}
+                  key={i}
+                >
+                  <PlusIcon className={styles.plusIcon} />
+                  <Markdown className={styles.plusDescription}>{plus}</Markdown>
+                </div>
+              ))}
```

Comment:
Semantic problem.
Here you should declare `markdown` type and simply put there a bullet list. Then in the `markdown.tsx` you should render proper styling.

Reply:
I don't think it's a semantic problem. I think it is a different solution to the problem that gives bigger control and better understading to person writing in sanity. I think both solutions are valid. What do you think?

Reply:
If you want to keep your current Sanity schema - no problem. But it is still a semantic mistake, so change displaying pluses as a `<li>` elements in the `<ul>` tag